dev/src/index.ts

```diff
@@ -243,6 +243,8 @@ const MAX_CONCURRENT_REQUESTS_PER_CLIENT = 100;
  * @property {Timestamp} lastUpdateTime The update time to enforce. If set,
  *  enforces that the document was last updated at lastUpdateTime. Fails the
  *  operation if the document was last updated at a different time.
+ * @property {boolean} exists If set, enforces that the target document must
+ * or must not exist. This property is only valid with delete operations.
```

Comment:
I would delete the last sentence. It can be used with delete(), even if it does not make a lot of sense. It could be used to trigger an error if the deleted document does not exist for example.

Reply:
Done.